src/segment_reshape/map_tool/segment_reshape_tool.py

```diff
@@ -158,6 +166,26 @@ def _handle_mouse_click_event(
                 success=True,
             )
 
+    def _handle_key_event(self, key: Qt.Key) -> None:
+        if self.reshape_mode is True:
+            if key == Qt.Key_Escape or (
+                key == Qt.Key_Backspace
```

Comment:
should this be also `key in (backspace, delete)` like the following `elif`?

Reply:
Ah, yes it should. Forgot to add delete option here.

I will fix this.